src/Fantomas.Tests/CommentTests.fs

```diff
@@ -4,6 +4,39 @@ open NUnit.Framework
 open FsUnit
 open Fantomas.Tests.TestHelper
 
+[<Test>]
+let ``comments on enums - https://github.com/fsprojects/fantomas/issues/572``() =
```

Comment:
Minor nitpick, I write `, 100` to reference an issue.
See https://github.com/fsprojects/fantomas/blob/master/src/Fantomas.Tests/CompilerDirectivesTests.fs#L365
I should write these little things down somewhere. It doesn't really matter all that much and it will just improve some constancy.

Reply:
sure let me correct that